code/modules/events/spontaneous_appendicitis.dm

```diff
@@ -0,0 +1,11 @@
+/datum/event/spontaneous_appendicitis
+	no_fake = 1
+
+/datum/event/spontaneous_appendicitis/start()
+	for(var/mob/living/carbon/human/H in shuffle(living_mob_list)) if(H.client && H.stat != DEAD)
```

Comment:
Might be a good idea to check for antag status here.
Could be difficult for a antag to get that resolved if it happens mid round

Reply:
Fixed.